pkgs/development/tools/sassc/default.nix

```diff
@@ -23,6 +23,7 @@ stdenv.mkDerivation rec {
     description = "A front-end for libsass";
     homepage = "https://github.com/sass/sassc/";
     license = licenses.mit;
+    mainProgram = pname;
```

Comment:
Not sure if this matters but `rg mainProgram` shows that most packages use string literals

Reply:
yeah thats true -- looks like something like 20 packages use pname including the one i picked at random xd.  I think it's equivalent, just a matter of convention?  I can change it if it would be better

Reply:
Changing for convention SGTM